### DISCUSSION TITLE
Which learning rate is displayed in the train log? Which code generates this log?

### DISCUSSION THREAD

**Discussion by susaha** [open-mmlab/mmdetection#8843] [General]
During training, we can see a training log. An example is shown below. I want to know about the lr, is it the lr of the backbone? Which part of the code generates this log?
2022-09-17 00:43:35,842 - mmdet - INFO - Iter [1700/90000]	lr: 1.000e-05, eta: 17:26:19, time: 0.759, data_time: 0.068, memory: 6854, loss_cls: 0.3668,...

**Comment by vansin** [comment]
which is the version of mmdet ?

**Reply by susaha** [reply]
This is the lr of the backbone.
